### ISSUE TITLE
Redundancy between DynVar 19 level and "day" 8 level variables

### ISSUE THREAD

**Issue by martinjuckes** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#63]
There is redundancy between daily pressure level fields requested by DynVar and in "day". The DynVar request is for 40 years of data on 19 levels. The CMIP request is for 65 years of data on 8 levels. It will be better to have 65 years of data on 19 levels than.

**Comment by taylor13** [comment]
Unless a group is participating in DynVar, we don't want to collect data on 19 levels.  In general we want to keep only daily data that is normally needed, and except for DynVar, there is no need for data at such a high vertical resolution.  

If the requested years need to be the same between DynVar and CMIP, then I suggest that they ask for their data for 65 years.  (This would be wasteful, however, because they only will look at 40 years.)

In summary, number of levels drq indicates are requested when one is participating in DynVar will be different from the number of levels when *not* participating in DynVar.

**Comment by martinjuckes** [comment]
Partially resolved in 01.00.12 (time slices for dynvar historical request ended), but need to encode choice options.
